### PR TITLE
test host_manager is running in the container

### PR DESCRIPTION
**Pull request by Yuhu-kth**

### CODE REVIEW

File: emulation-system/tests/test_start_host_manager.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    emulation_env_config.host_manager_config.host_manager_log_dir = "/var/log/host_manager"
+    emulation_env_config.host_manager_config.host_manager_log_file = "host_manager.log"
+    emulation_env_config.host_manager_config.host_manager_max_workers = 4
+    # Create a mock logger
+    logger = logging.getLogger("test_logger")
+    ip = "15.15.15.15"
+    try:
+        container_setup.reload()
+        assert container_setup.status == "running", f"Container {container_setup.name} is not running"
+        print(f"Container {container_setup} is running.")
```
Comment (Limmen) [inline emulation-system/tests/test_start_host_manager.py:131]: Can remove this print

**Review by Limmen — CHANGES_REQUESTED** [review]
Looks good, one print statement to remove

File: emulation-system/tests/test_start_host_manager.py (hunk 1 above)
Comment (Yuhu-kth) [reply emulation-system/tests/test_start_host_manager.py:131]: removed

**Comment by Yuhu-kth** [comment]
For the start_host_manager test, 30 containers in total, 29 passed, 1 didn't pass because its not running, don't know why it's not running yet

**Review by Limmen — APPROVED** [review]

**Pull request merged**